### DISCUSSION TITLE
WARNING: Caught will_solve in state None, but failed to capture its initialization

### DISCUSSION THREAD

**Discussion by christiankakesa** [trailofbits/manticore#2499] [Q&A]
When running manticore against my contract I have this warning: `WARNING: Caught will_solve in state None, but failed to capture its initialization` 

How can I solve this warning?
Where to find documentation about this kind of error?
Could this error be ignored?

![image](https://user-images.githubusercontent.com/336317/140233713-f2b1ee3f-c5f7-43cd-bdef-2bb6c5e08ec2.png)

**Comment by superical** [comment]
@fenicks Did you manage to find any info about this warning?

**Reply by christiankakesa** [reply]
Hi,
I solved it partially by upgrading from Ubuntu 18.04 to 20.04 and python lib like protobuf.
But now I'm dealing with timeout issues :disappointed:
